packages/api-gateway/src/drivers/http/plugins/services.js

```diff
@@ -1,12 +1,14 @@
 'use strict';
 
-const MathServices = require('@booking-services/math');
+//const MathServices = require('@booking-services/math');
+const GeolocationServices = require('@booking-services/geolocation');
 
 const PluginLoader = require('fastify-plugin');
 
 async function services(fastify) {
 
-	await fastify.decorate('mathServices', MathServices)
+	//await fastify.decorate('mathServices', MathServices)
```

Comment:
@CETHER what so you think if we keep the math package instead of removing it or comment it just to have a functional package that we can use as base.

Reply:
Ok, I will keep the math paccakge to have that as base